constitution.tex

```diff
@@ -794,6 +794,7 @@
 	\item The candidates will be notified of their nomination.
 		Each candidate is given a minimum of twenty-four hour period to accept or decline the nomination.
 		A list of all nominees who have accepted their nominations will be posted shortly thereafter.
+	\item The date and time of the candidate(s)'s speeches will be announced by a member of the Executive Board at least five (5) days prior to the event. This will be treated as a House Meeting, with the expectation that all Active Members will attend and attendance will be taken. If the speeches take place at a traditional House Meeting, only one attendance will be taken.
```

Comment:
I'd also like to point out, house meetings are at an established time and are therefore announced (in effect) months in advance. 5 days isn't necessarily enough notice to clear a schedule for a saturday or sunday morning. This is part of why we generally do elections at house meetings, excepting the large scale elections at the end of the year, which generally have significant notice.

I'd also wonder, what's the goal of requiring attendance? Quorum isn't required for speeches, and this attendance requirement has minimal enforcement.

Reply:
Active members should be involved in this process, as this is probably as important as anything that would be discussed at house meeting. Everyone should be making an effort to be there to make sure the best candidate is selected


Also if we need overrides at the speeches it would be much less of a rush, altho doing it as a form beforehand is likely an easier option

Reply:
note: Overrides don't need to happen at speeches, they just need to happen before votes are counted, which means that most of the time they can occur at house meeting, when quorum is far easier to obtain.

I think whether or not this change goes through, the Chairperson docs should note that elections should be during house meetings where possible,